osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -163,16 +175,36 @@ private void load(OverlayColourProvider colourProvider)
                                                     DocumentMargin = new MarginPadding(0),
                                                     DocumentPadding = new MarginPadding(0),
                                                 },
-                                                info = new FillFlowContainer
+                                                new FillFlowContainer
                                                 {
                                                     AutoSizeAxes = Axes.Both,
                                                     Direction = FillDirection.Horizontal,
                                                     Spacing = new Vector2(10, 0),
                                                     Children = new Drawable[]
                                                     {
-                                                        new DrawableDate(Comment.CreatedAt, 12, false)
+                                                        info = new FillFlowContainer
+                                                        {
+                                                            AutoSizeAxes = Axes.Both,
+                                                            Direction = FillDirection.Horizontal,
+                                                            Spacing = new Vector2(10, 0),
+                                                            Children = new Drawable[]
+                                                            {
+                                                                new DrawableDate(Comment.CreatedAt, 12, false)
+                                                                {
+                                                                    Colour = colourProvider.Foreground1
+                                                                }
+                                                            }
+                                                        },
+                                                        actionsContainer = new LinkFlowContainer(s => s.Font = OsuFont.GetFont(size: 12, weight: FontWeight.Bold))
                                                         {
-                                                            Colour = colourProvider.Foreground1
+                                                            AutoSizeAxes = Axes.Both,
+                                                            Spacing = new Vector2(10, 0)
+                                                        },
+                                                        actionsLoading = new LoadingSpinner
```

Comment:
In the tests you've added, there's no way to see this in action.

Can you add a delay to the test request completion so it is visible? Would also be nice to have a test testing the failure scenario.

Reply:
Dummy API handling seems to be done in update thread.
When i'm writing 
```
Thread.Sleep(1000);
req.TriggerSuccess(cb);
return true;
```
the whole test browser is hanging up for a second, not request itself. Any tips how i can work around this?

Reply:
I've updated the test to show how this can be done!